pxr/usdImaging/usdImagingGL/CMakeLists.txt

```diff
@@ -142,3 +146,4432 @@ pxr_build_test(testUsdImagingGLSurfaceShader
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -frameAll -offscreen -lighting -times 1 3 4 5 4 -stage bunnyCurves.usda -write testUsdImagingGLBasisCurvesVaryingTopology.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLBasisCurvesVaryingTopology_003.png
+        testUsdImagingGLBasisCurvesVaryingTopology_004.png
+        testUsdImagingGLBasisCurvesVaryingTopology_005.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLBasisCurvesVaryingTopology
+)
+
+pxr_register_test(testUsdImagingGLAnimatedCamera
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -lighting -frameAll -camera /World/main_cam -times 1 2 3 -stage cameraMoving.usda -write testUsdImagingGLAnimatedCamera.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLAnimatedCamera_001.png
+        testUsdImagingGLAnimatedCamera_002.png
+        testUsdImagingGLAnimatedCamera_003.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLAnimatedCamera
+)
+
+pxr_register_test(testUsdImagingGLAnimatedLights
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -lighting -frameAll -times 1 2 3 -stage lightMoving.usda -write testUsdImagingGLAnimatedLights.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLAnimatedLights_001.png
+        testUsdImagingGLAnimatedLights_002.png
+        testUsdImagingGLAnimatedLights_003.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLAnimatedLights
+)
+
+pxr_register_test(testUsdImagingGLSdr_1
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -lighting -frameAll -stage object.usda -write testUsdImagingGLSdr1.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLSdr1.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLSdr
+)
+
+pxr_register_test(testUsdImagingGLSdr_2
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -lighting -frameAll -stage usdGlslfxEmbedded.usda -write testUsdImagingGLSdr2.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLSdr2.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLSdr
+)
+
+pxr_register_test(testUsdImagingGLInvalidInfoId
+    COMMAND "${CMAKE_INSTALL_PREFIX}/tests/testUsdImagingGLBasicDrawing -offscreen -lighting -frameAll -stage invalid_infoid.usda -write testUsdImagingGLInvalidInfoId1.png"
+    IMAGE_DIFF_COMPARE
+        testUsdImagingGLInvalidInfoId1.png
+    FAIL 1
+    FAIL_PERCENT 1
+    PERCEPTUAL
+    EXPECTED_RETURN_CODE 0
+    TESTENV testUsdImagingGLInvalidInfoId
+)
+
+# NOTE: These Pass if we allow the perceptual threshold to a fail percent of 2 vs 1
+# From the testSpecs.xml these thresholds are set to 1 so there might be some differences
+# between idiff -p vs perceptualdiff
```

Comment:
Another thing to point out but I think the small increase to 2 percent is acceptable